testapi.pm

```diff
@@ -1252,25 +1252,27 @@ sub type_string {
         return;
     }
 
-    my $max_interval = $args{max_interval}       // 250;
-    my $wait         = $args{wait_screen_change} // 0;
-    my $wait_still   = $args{wait_still_screen}  // 0;
-    bmwqemu::log_call(string => $log, max_interval => $max_interval, wait_screen_changes => $wait, wait_still_screen => $wait_still);
+    my $max_interval   = $args{max_interval}       // 250;
+    my $wait           = $args{wait_screen_change} // 0;
+    my $wait_still     = $args{wait_still_screen}  // 0;
+    my $wait_timeout   = $args{timeout}            // 30;
+    my $wait_sim_level = $args{similarity_level}   // 47;
```

Comment:
Could you please, also update perldoc above for your changes, so it's visible for other who wants to use this feature.

Reply:
Done, please check!

Reply:
looks good!